### DISCUSSION TITLE
[Black Box - Privacy Enhanced] Is there a way to extend past 2GB for the Black Box?

### DISCUSSION THREAD

**Discussion by TFWol** [sandboxie-plus/Sandboxie#4058] [Q&A Feedback] [answered]
App's have started breaking and complaining about free space in the black box. Does that have something to do with the 2GB image file? If so, how do I extend this?

**Comment by offhub** [comment, marked as answer]
Sandboxie Plus > Sandbox > Maintenance > Virtual Disks > Select the disk > Extend size... (Don't forget to read the texts in the Extend dialog!)

**Reply by TFWol** [reply]
Thank you so much for the quick reply. That solved the problem.

Is there a link in the Repo / Docs / Forum that points this out?

I couldn't find anything with the keywords I used.

**Reply by offhub** [reply]
This is a feature of the ImDisk Toolkit, so there is no documentation that I know of.
